### PR TITLE
Added flp project (with README.md) and initial redirections specifica…

### PR DESCRIPTION
**Pull request by magrod**
…tions

Author: Diego Magro (diego.magro@unito.it). 
Organization: Dipartimento di Informatica, Università di Torino

BR,
diego magro

### CODE REVIEW

**Comment by davidlehn** [comment]
Could you please add maintainer github ids to the README?  Thanks.

File: flp/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+# flp
+A pool of ontologies for representing data about films and film contents
+
+##Contact:
```
Comment (davidlehn) [inline flp/README.md:5]:
```suggestion
## Contact:
```

File: flp/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+This space is administered by:
+
+**Diego Magro**
```
Comment (davidlehn) [inline flp/README.md:8]: Can you fix the render of the lines here?  Either make it a list or add trailing spaces to force line breaks.  Thanks.

File: flp/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+*Researcher*
+Dipartimento di Informatica, Università di Torino,
+Turin, Italy
+<diego.magro@unito.it>
+GitHub: [magrod](https://github.com/magrod)
```
[the remark this answers is not in the document]
Comment (magrod) [reply flp/README.md]: Done!
Thanks!

Best regards,
diego magro

**Pull request merged**